seichi-onp-k8s/manifests/seichi-kubernetes/app-templates/seichi-debug-minecraft-on-seichiassist-pr/templates/debug-s1/stateful-set.yaml

```
@@ -33,6 +33,27 @@ spec:
             - "sh"
             - "-c"
             - 'wget -O /root/jmx-exporter-download/jmx-exporter-javaagent.jar "${JMX_EXPORTER_URL}"'
+        - name: mod-downloader
+          image: ghcr.io/giganticminecraft/mod-downloader:sha-8326d06
+          env:
+            - name: MINIO_ENDPOINT
+              value: seichi-private-plugin-blackhole-minio-console.minio:9001
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: mcserver--common--config-secrets
+                  key: MINIO_DEBUG_ACCESS_KEY
+            - name: MINIO_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: mcserver--common--config-secrets
+                  key: MINIO_DEBUG_ACCESS_SECRET
```

```suggestion
            - name: MINIO_ACCESS_KEY
              valueFrom:
                secretKeyRef:
                  name: mcserver--common--config-secrets
                  key: MINIO_DEBUG_ACCESS_KEY
```

同じのが重複しちゃってそう？一つで大丈夫です

---

`name`が`MINIO_ACCESS_SECRET`になっているべきでした。
コピペミスです、直します